test/controllers/EditSetControllerSpec.scala

```diff
@@ -3277,9 +3230,9 @@ class EditSetControllerSpec extends BaseSpec {
   ): Assertion = {
     document must haveTitle(expectedEditRecordPage.title)
     document must haveCaption(expectedEditRecordPage.caption)
-    document must haveActionButtons(expectedEditRecordPage.button.value, expectedEditRecordPage.button.label)
-    document must haveSelectionForOrderingField(expectedEditRecordPage.expectedOptionsForField)
-    document must haveSelectionForOrderingDirection(expectedEditRecordPage.expectedOptionsForDirection)
+    expectedEditRecordPage.tableHeaders.foreach { header =>
```

Comment:
Perhaps we should also assert that there are 3 of them? 

Or, add three fields to ExpectedEditSetPage. 

As it stands, we could forget to include some or all of the fields.

Personally, I'd go with three fields, like: `tableHeaderForCCR`, etc ... then we are forced to declare what we expect in each case.

Reply:
Confirmed.